install_config/install/advanced_install.adoc

```diff
@@ -125,6 +125,13 @@ link:../../install_config/install/rpm_vs_containerized.html[RPM vs
 Containerized] for more details.
 ifdef::openshift-enterprise[]
 Containerized installations are supported starting in OSE 3.1.1.
+
```

Comment:
yes, move endif; this config applies to all distros.

i don't think you want to take the space for a real description of these params here, however it would be a good place to link to the relevant docker docs. https://docs.docker.com/engine/admin/logging/overview/#json-file-options

For the driver (https://docs.docker.com/engine/admin/logging/overview/#configure-logging-drivers) I'm not entirely sure we should mention this; we only support the json-file driver at this time, as using any other driver makes those logs un-retrievable by ordinary openshift mechanisms and they would need their own logging system). So any value other than `json-file` is going to be a *really* advanced install.

It *would* be a good idea to find out what format is expected for `docker_log_options`. The docs describe them as command-line opts, so you end up with e.g. `--log-opt max-size=10m --log-opt max-file=5` however I doubt that's the format of the ansible config value - would have to ask whoever implemented it.